src/.clang-format

```diff
@@ -26,7 +28,6 @@ IndentCaseLabels: false
 IndentFunctionDeclarationAfterType: false
 IndentWidth:     4
 KeepEmptyLinesAtTheStartOfBlocks: false
-Language:        Cpp
```

Comment:
What's the effect of removing Cpp as language? Does it autodetect c++ or is it the default in all recent clang-format versions?